test/oauth-server/service/token.go

```diff
@@ -57,8 +56,14 @@ func makeAccessToken(clientID, host, deviceID, scopes string, issuedAt, expires
 		return nil, fmt.Errorf("failed to set %v: %w", jwt.IssuerKey, err)
 	}
 	if deviceID != "" {
-		if err := token.Set(TokenDeviceID, deviceID); err != nil {
-			return nil, fmt.Errorf("failed to set %v: %w", TokenDeviceID, err)
+		if err := token.Set(uri.DeviceIDClaimKey, deviceID); err != nil {
+			return nil, fmt.Errorf("failed to set %v: %w", deviceID, err)
```

Comment:
instead uri.DeviceIDClaimKey of deviceID, key will tell you more than a uuid; or maybe log both

Reply:
fixed